.env

```diff
@@ -1,7 +1,7 @@
 ACCESS_TOKEN_SECRET = jsfgfjguwrg8783wgbjs849h2fu3cnsvh8wyr8fhwfvi2g225
 REFRESH_TOKEN_SECRET = 825y8i3hnfjmsbv7gwajbl7fobqrjfvbs7gbfj2q3bgh8f42
 
-TEST_TOKEN= eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjE4LCJlbWFpbCI6ImFAZ21haWwuY29tIiwiaWF0IjoxNjcwNzU1MzM2LCJleHAiOjE2NzA3NTYyMzZ9.RUu9kwa9sJusJV9c6bN8xGOuDmOBLvOeAEOp5xx0n2c
+TEST_TOKEN= eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjIyLCJlbWFpbCI6ImFiQGdtYWlsLmNvbSIsImlhdCI6MTY3MDgzNzM1MywiZXhwIjoxNjcwOTIzNzUzfQ.ICxWakfKoTgbB4QJy703vU4JBzSXVuh7gVZmNybbfTs
 
 port = 5000
 DB = lab1
```

Comment:
.env file should not be added to the git.
You should ignore this file in `.gitignore` file.

Reply:
because it contains secret information

Reply:
Added to gitignore